install_config/install/advanced_install.adoc

```diff
@@ -1362,10 +1362,53 @@ specifications, and save it as *_/etc/ansible/hosts_*.
 [[running-the-advanced-installation]]
 == Running the Advanced Installation
 
-After you have xref:configuring-ansible[configured Ansible] by defining an
-inventory file in *_/etc/ansible/hosts_*, you can run the advanced installation
-using the following playbook:
+After you have finished xref:configuring-ansible[configuring Ansible] by
+defining your own inventory file in *_/etc/ansible/hosts_* or modifying one of
+the xref:adv-install-example-inventory-files[example inventories], follow these
+steps to run the advanced installation:
 
+// tag::BZ1466783-workaround-install[]
+. In {product-title}
+ifdef::openshift-enterprise[]
+3.4,
+endif::[]
+ifdef::openshift-origin[]
+1.4,
+endif::[]
+the master connected to the etcd cluster using the host name of the etcd
+endpoint. In {product-title}
+ifdef::openshift-enterprise[]
+3.5,
+endif::[]
+ifdef::openshift-origin[]
+1.5,
+endif::[]
+the master now connects to etcd via IP address.
++
```

Comment:
Would it be better if this information were in paragraph form than numbered list? Those first two sentences in Step 1 confused me a bit as they are not steps. Maybe something like:
http://pastebin.test.redhat.com/500963